orbs.rb

```diff
@@ -0,0 +1,23 @@
+require "sinatra"
+require 'sinatra/json'
```

Comment:
I am okay with either single quotes or double quotes, but I strongly prefer that only one is used throughout the entire project.